custom-words.txt

```diff
@@ -1151,6 +1154,7 @@ MSAZRDE
 MSAZRUSGO
 MSCONCAT
 mscv
+msdb
```

Comment:
which context do we need to expose msdb, which is a strange name? can we change it?